frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -47,12 +47,7 @@ export const HeadersSync: FunctionComponent<Props> = ({ coinCode }) => {
   const total = status.targetHeight - status.tipAtInitTime;
   const value = 100 * (status.tip - status.tipAtInitTime) / total;
   const loaded = !total || value >= 100;
-  let formatted = status.tip.toString();
-  let position = formatted.length - 3;
-  while (position > 0) {
-    formatted = formatted.slice(0, position) + '\'' + formatted.slice(position);
-    position = position - 3;
-  }
+  const formatted = new Intl.NumberFormat(i18n.language).format(status.tip);
 
```

Comment:
thanks @ide  ❤️ 

I put this in a separate PR https://github.com/digitalbitbox/bitbox-wallet-app/pull/1730, but I am not sure how to make you co-author in git.


https://github.com/digitalbitbox/bitbox-wallet-app/pull/1730

Reply:
No worries =) Glad I could be of some help